extensions/ql-vscode/test/vscode-tests/cli-integration/debugger/debugger.test.ts

```diff
@@ -177,4 +177,27 @@ describeWithCodeQL()("Debugger", () => {
       expect(editor.document.isDirty).toBe(false);
     });
   });
+
+  it("should pass additionalArgs through to query server", async () => {
+    await withDebugController(appCommands, async (controller) => {
+      await controller.startDebugging(
+        {
+          query: quickEvalQueryPath,
+          additionalArgs: {
+            // Overrides the value passed to the query server
+            queryPath: simpleQueryPath,
```

Comment:
Oh...interesting. This will allow you to override any args set by the extension. Could be dangerous, but then again it's an internal, undocumented feature.

Reply:
Yeah. There are a few sharp edges.